.github/workflows/ci.yml

```diff
@@ -178,11 +177,21 @@ jobs:
     # - MOODLE_XX_STABLE
     # - MOODLE_XXX_STABLE
     # - main
-    if: needs.pre_job.outputs.should_skip != 'true' && needs.prepare_matrix.outputs.release_required == 'true' && secrets.MOODLE_ORG_TOKEN != ''
+    if: needs.pre_job.outputs.should_skip != 'true' && needs.prepare_matrix.outputs.release_required == 'true'
     runs-on: 'ubuntu-latest'
+    outputs:
+      has-secrets: ${{ steps.check-secrets.outputs.has-secrets }}
     steps:
+      - name: Check if MOODLE_ORG_TOKEN has been supplied
+        id: check-secrets
+        env:
+          SECRET_TO_CHECK: '${{ secrets.moodle_org_token }}'
+        if: ${{ env.SECRET_TO_CHECK != '' }}
+        run: echo "::set-output name=has-secrets::true"
+
       - name: Run plugin release
+        if: steps.check-secrets.outputs.has-secrets != ''
         uses: catalyst/catalyst-moodle-workflows/.github/plugin/release@main
         with:
           plugin_name: ${{ needs.prepare_matrix.outputs.component }}
-          moodle_org_token: ${{ secrets.MOODLE_ORG_TOKEN }}
+          moodle_org_token: ${{ secrets.moodle_org_token }}
```

Comment:
Why the change in case?

Reply:
It's using the locally scoped secret I believe. This is referenced at the top of the workflow file, and in other examples I've seen. I think this might have been something I've missed initially when I added it.

The uppercase one refers to the secret coming out from the org / plugin settings, so that again, is different.

Reply:
Further backed up by the usage from the plugin repo + readme docs:

![image](https://user-images.githubusercontent.com/9924643/174233341-97200ff6-c1bf-4def-9a64-7e217b25c68d.png)

Source: https://github.com/catalyst/moodle-tool_crawler/pull/161/files